### PR TITLE
responsive profile

### PR DESCRIPTION
**Pull request by kziemianek**
##### Description
profile view resuscitation

##### Checklist
- [x] linter status: 100% pass
- [ ] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
front-end

##### Testing
locally

##### Refers/Fixes
Fixes: #817

### CODE REVIEW

**Comment by kziemianek** [comment]
Hope this is enough untill revamp.

iphone 7:
iphone_7

ipad:
ipad

desktop:
desktop

File: app/dashboard/templates/profile_details.html [hunk 1]
```diff
@@ -1,224 +1,182 @@
...
+                                    </div>
+                                    {% endfor %} {% endif %}
+                                </div>
                             </div>
                             <div>
-                            <h4>Repos</h4>
+                                <h4>Repos</h4>
                                 <ul class="repos">
                                     {% for r in profile.repos_data %}
-                                        <li>
```
Comment (thelostone-mc) [inline app/dashboard/templates/profile_details.html]: Don't the other files follow this ? It's easier to read when we conditional statements

File: app/dashboard/templates/profile_details.html [hunk 2]
```diff
@@ -1,224 +1,182 @@
...
+                                            <strong>{% trans "Details" %}</strong>
+                                        </div>
+                                        <div class="col-4 col-md-4  d-none d-md-block">
+                                            <strong>{% trans "Amount" %}</strong>
+                                        </div>
+                                    </div>
+                                    {% for b in bounties %}
+                                    <div class="row bounty-row" data-ref-url="{{b.url}}">
+                                        <div class="col-2 d-none d-sm-block">
+                                            {% if profile.handle == b.fulfiller_github_username %} {% trans "Worker" %} {% elif profile.handle == b.bounty_owner_github_username%} {% trans "Funder" %} {% elif profile.handle == b.org_name %} {% trans "Repo
```
Comment (thelostone-mc) [inline app/dashboard/templates/profile_details.html]: Could we break this into multiple lines ! In one line I'm forced to scroll during code reviews :P .

File: app/dashboard/templates/profile_details.html [hunk 3]
```diff
@@ -230,19 +188,22 @@ <h6>
...
 
-</html>
+<script>
+    $('.tip-row, .bounty-row').click(function () {
+        window.location = $(this).data('ref-url')
+    })
+</script>
+
+
+</html>
```
Comment (thelostone-mc) [inline app/dashboard/templates/profile_details.html]: Could we reindent the whole file ? With 2 spaces.

File: app/dashboard/templates/profile_details.html [hunk 4]
```diff
@@ -230,19 +188,22 @@ <h6>
...
 
 
-
 <!-- jQuery -->
 <script src="{% static "v2/js/abi.js" %}"></script>
 <script src="{% static "v2/js/tokens.js" %}"></script>
 <script src="{% static "v2/js/shared.js" %}"></script>
 
-</html>
+<script>
```
Comment (thelostone-mc) [inline app/dashboard/templates/profile_details.html]: This should be indented by one level as it's enclosed within the HTML tag

File: app/dashboard/templates/profile_details.html [hunk 5]
```diff
@@ -230,19 +188,22 @@ <h6>
         </div>
 
 
-    {% include 'shared/bottom_notification.html' %}
-    {% include 'shared/analytics.html' %}
-    {% include 'shared/footer_scripts.html' %}
-    {% include 'shared/rollbar.html' %}
-    {% include 'shared/footer.html' %}
+        {% include 'shared/bottom_notification.html' %} {% include 'shared/analytics.html' %} {% include 'shared/footer_scripts.html'%}
+        {% include 'shared/rollbar.html' %} {% include 'shared/footer.html' %}
```
Comment (thelostone-mc) [inline app/dashboard/templates/profile_details.html]: Stick to 2 spaces 
And let's keep each of them in their own line ^_^
Easier to read them that way

File: app/dashboard/templates/profile_details.html [hunk 6]
```diff
@@ -1,224 +1,182 @@
...
-                                        <li>
-                                            {% trans "No repos" %}
-                                        </li>
+                                            {% if r.description %}
+                                            <p class="mb-1">{{r.description}}</p>
+                                            {% endif %}
+                                        </a>
+                                    </li>
+                                    {% endfor %} {% if profile.repos_data|length == 0 %}
+                                    <li>
```
Comment (thelostone-mc) [inline app/dashboard/templates/profile_details.html]: Break into multiple lines

File: app/dashboard/templates/profile_details.html [hunk 7]
```diff
@@ -1,224 +1,182 @@
...
                                 <ul class="repos">
                                     {% for r in profile.repos_data %}
-                                        <li>
-                                            <div class="dot">&nbsp;</div>
-                                            <a href="{% url "explorer" %}?q={{r.html_url}}">
-                                                <p class="meta mr-3">{% blocktrans %}{{r.stargazers_count}} stars{% endblocktrans %}</p>
-                                                <h6>
+                                    <li>
+                                        <div class="dot">&nbsp;</div>
+                                        <a href="{% url " explorer " %}?q={{r.html_url}}">
```
Comment (thelostone-mc) [inline app/dashboard/templates/profile_details.html]: Do we need the extra space? :sweat_smile:

**Comment by thelostone-mc** [comment]
Could we break down the expressions into multiple lines for better readability  ?
Example : [here](https://github.com/gitcoinco/web/pull/825/files#diff-6651d89ad91df47d8a4ccf0cf8e5c2d7R119)  [there are more]

File: app/dashboard/templates/profile_details.html [hunk 8]
```diff
@@ -1,248 +1,210 @@
...
 <!DOCTYPE html>
 <html lang="en">
 
-<head>
-    {% include 'shared/head.html' %}
-    {% include 'shared/cards_pic.html' %}
-</head>
-<body class="interior {{active}}">
+  <head>
+    {% include 'shared/head.html' %} {% include 'shared/cards_pic.html' %}
```
Comment (mbeacom) [inline app/dashboard/templates/profile_details.html]: As @thelostone-mc was saying, can we drop the second `{% include %}` to it's own line?

File: app/dashboard/templates/profile_details.html [hunk 9]
```diff
@@ -1,248 +1,210 @@
-{% load i18n email_obfuscator %}
-{% load humanize %}
-{% load static %}
+{% load i18n email_obfuscator %} {% load humanize %} {% load static %}
```
Comment (mbeacom) [inline app/dashboard/templates/profile_details.html]: We should either consolidate these or break them out into their own lines.

File: app/dashboard/templates/profile_details.html [hunk 10]
```diff
@@ -1,248 +1,210 @@
...
+            <img class="avatar" src="{{profile.local_avatar_url}}">
+            <h2 class="mt-3">
+              Profile: {{profile.handle}}
+              <span id="alpha">Alpha</span>
+            </h2>
+            {% if profile.data.bio %}
+            <p>
+              {{profile.data.bio}}
+            </p>
+            {% endif %} {% if profile.authors|length %}
```
Comment (mbeacom) [inline app/dashboard/templates/profile_details.html]: Can you drop the `if` to the next line?

File: app/dashboard/templates/profile_details.html [hunk 11]
```diff
@@ -1,248 +1,210 @@
...
-                                            {% trans "No repos" %}
-                                        </li>
-                                    {% endif %}
-                                </ul>
-                            </div>
-                        </div>
-                        {% include 'shared/newsletter.html' %}
+                      </div>
+                    </div>
+                    {% endfor %} {% endif %}
```
Comment (mbeacom) [inline app/dashboard/templates/profile_details.html]: Can we drop down all of the` {% endfor %} {% endif %}` throughout?

**Comment by kziemianek** [comment]
My bad, this is how my formatter formated the file.
Now I formatted in manually. 
Can anybody recomend any good vsc plugin which will format html files with these django tags properly ?

**Comment by mbeacom** [comment]
@kziemianek Can you take a look at the profile page?  
screenshot 2018-04-09 14 29 26

**Comment by mbeacom** [comment]
@kziemianek Regarding VSCode formatting... your issue is related to the default HTML formatter baked into vsc.  https://stackoverflow.com/a/42579476/2560922 - Disabling it and/or defining custom rules using beautifier is typically what people do.

**Comment by kziemianek** [comment]
@mbeacom , solved.

**Comment by mbeacom** [comment]
@kziemianek Thanks for the contribution!

**Pull request merged**